docker-compose.yml

```diff
@@ -215,6 +219,12 @@ services:
     image: zookeeper:3.4
     restart: always
     hostname: zoo2
+    expose:
+      - "3888"
+    ports:
+      - "2182:2181"
```

Comment:
Shouldn't this be 2181:2181?
If zoo1 fails clients of the zookeepers probably try to connect to 2181 of other zookeepers

Reply:
I think these ports(2182-2183) was for local machine testing.
So I'll change this code `2182:2181` to `2181`